de.dlr.sc.virsat.model.extension.requirements.feature/build.properties

```diff
@@ -0,0 +1,2 @@
+bin.includes = feature.xml,\

```

Comment:
Shouldnt there also be a feature.properties file?
Its usually needed for the shared license to work.